packages/kv-store/src/keyv/keyv.ts

```diff
@@ -156,6 +158,39 @@ export class Keyv<Value = any> extends EventEmitter implements KeyvStore<Value>
       .then(() => Promise.all(results))
   }
 
+  private isMapWithEntries(store: any): store is Map<string, Value> {
+    return store instanceof Map && typeof store.entries === 'function'
+  }
+
+  private async getFromMapWithPrefix(
+    prefixes: Array<string>,
+    options?: {
```

Comment:
should we use this?

Reply:
removed, this started as a copy from get()